src/mygrad/random/random/rand.py

```diff
@@ -0,0 +1,27 @@
+from mygrad import Tensor
+import numpy as np
+
+
+def rand(*shape, constant=False):
```

Comment:
There is too much space between these docstring sections. There should only be one blank line separating the sections - could you make sure that the docstrings adhere to this?